File: LanguAI.Backend/LanguAI.Backend/ViewModels/Interaction/CommentViewModel.cs
using LanguAI.Backend.ViewModels.User;

namespace LanguAI.Backend.ViewModels.Interaction;

public class CommentViewModel
{
    public int Id { get; set; }
    public UserViewModel User {get; set;}
    public int UserId { get; set; }
    public DateTime Created { get; set; }
    public bool Liked { get; set; }
    public int NumberOfLikes { get; set; }
    public string Text { get; set; }
}

File: LanguAI.Frontend/src/api/models/comment-view-model.ts
/* tslint:disable */
/* eslint-disable */
import { UserViewModel } from '../models/user-view-model';
export interface CommentViewModel {
  created?: string;
  id?: number;
  liked?: boolean;
  numberOfLikes?: number;
  text?: string | null;
  user?: UserViewModel;
  userId?: number;
}
